src/middlewared/middlewared/plugins/apps/ix_apps/lifecycle.py

```diff
@@ -18,7 +19,8 @@ def get_rendered_template_config_of_app(app_name: str, version: str) -> dict:
     for rendered_file in get_rendered_templates_of_app(app_name, version):
         with contextlib.suppress(FileNotFoundError, yaml.YAMLError):
             with open(rendered_file, 'r') as f:
-                rendered_config.update(yaml.safe_load(f.read()))
+                if (data := yaml.safe_load(f.read())) is not None:
+                    rendered_config.update(data)
```

Comment:
Both the original and updated code here uses `yaml.safe_load(f.read())`.  OTOH, there's a bunch of code that does in this PR that does `yaml.safe_load(f)`.  Any reason to not be consistent?

Reply:
Agree. My preference is to pass the file object instead of doing `f.read()`